File: API/Models/Live.cs
using Microsoft.VisualBasic;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ABrechozeiraApp.Models;

public partial class Live
{
    public int Id { get; set; }
    [StringLength(50)]
    public string Titulo { get; set; } = null!;
    
    public string? Observacoes { get; set; }

    public DateTime DataLive { get; set; }

    public DateTime DataAlteracao { get; set; }

    [ForeignKey("Usuario")]
    public int UsuarioModificacaoId { get; set; }
    public Usuario? UsuarioModificacao { get; set; }
}

File: FRONT/src/app/modules/lives/models/live.ts
export interface Live {
  id: number;
  titulo: string;
  observacoes: string;
  dataLive: string;
  dataAlteracao: string;
  usuarioModificacaoId: number;
  usuarioModificacao: string;
  googleSheetUrl?: string;
}

export interface LiveCreate {
  titulo: string;
  observacoes: string;
  dataLive: string;
  usuarioModificacaoId: number;
  googleSheetUrl?: string;
} 